From c0671533cb5085e2e2637928f9b18d27ec9014a1 Mon Sep 17 00:00:00 2001
Message-ID: <c0671533cb5085e2e2637928f9b18d27ec9014a1.1729453324.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1729453324.git.sharlatanus@gmail.com>
References: <cover.1729453324.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 7 Oct 2024 20:42:16 +0100
Subject: [PATCH 20/42] gnu: python-regions: Update to 0.10.

* gnu/packages/astronomy.scm (python-regions): Update to 0.10.
[native-inputs]: Add python-setuptools.

Change-Id: Ie5fb9464af8c00570afbb78392f917b6f6c978fe
---
 gnu/packages/astronomy.scm | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 8929809e3a..2bd1e834cb 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3980,18 +3980,18 @@ (define-public python-pyvo
 (define-public python-regions
   (package
     (name "python-regions")
-    (version "0.9")
+    (version "0.10")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "regions" version))
        (sha256
-        (base32 "0kvfdzqry3vcvphd7ldmppbgn3ab97hbfzwxfrlxls92yi41h3i6"))))
+        (base32 "10cswrknj3qh9i1daynlx4ild66lwcyra5rs03h8s9j4l275274n"))))
     (build-system pyproject-build-system)
     (arguments
      (list
       #:test-flags
-      #~(list "-n" "auto")
+      #~(list "--numprocesses" "auto")
       #:phases
       #~(modify-phases %standard-phases
           ;; setup.py was removed in 84c80a280431adda00641cda5264c7de18b43b2f
@@ -4024,6 +4024,7 @@ (define-public python-regions
            python-pytest-astropy
            python-pytest-runner
            python-pytest-xdist
+           python-setuptools
            python-setuptools-scm))
     (home-page "https://github.com/astropy/regions")
     (synopsis "Package for region handling")
-- 
2.46.0

